Sept 5: Standard edition shift (#14336)
newrelic/docs-website: src/content/docs/style-guide/word-choice/pricing-language-guidelines.mdx

* We explain billing impacts in the main pricing and billing-related docs. The main docs in this area are:
  * [Usage-based pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/)
  * [Original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing)

> * When a feature requires a specific [pricing edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, this is an infrequent need.

Reviewer: ```suggestion
* When a feature requires a specific [pricing edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions), we include that in the requirements of the doc for that feature. But because there are only a few capabilities limited by the Pro and Enterprise editions, you won't see many of these callouts.
```

  * When referencing a model, it's good practice to point to either the doc for that model, or else the [doc explaining the differences between the models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-user-table), whichever makes more sense.
* "usage-based." Our newer pricing model can be referred to as “usage-based” or “consumption-based,” but “usage-based” is the most commonly used and more preferred term.
* Editions:

>   * Our [pricing editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are not "plans" or "models" or "tiers"; they are "editions". One exception is that it's acceptable to refer to the Free edition as the "free tier" simply because we have historically used that language a lot. 

Reviewer: ```suggestion
  * Our [pricing editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are not "plans" or "models" or "tiers"; they are "editions". One exception is that it's acceptable to refer to the Free edition as the "free tier" simply because we have used that label in the past. 
```
